src/Symfony/Component/Templating/Loader/CacheLoader.php

```diff
@@ -69,8 +69,8 @@ public function load(TemplateReferenceInterface $template)
 
         $content = $storage->getContent();
 
-        if (!is_dir($dir)) {
-            mkdir($dir, 0777, true);
+        if (!is_dir($dir) && !@mkdir($dir, 0777, true) && !is_dir($dir)) {
+            throw new \RuntimeException(sprintf('Cache Loader was not able to create a directory: %s', $dir));
```

Comment:
sometimes you have `The` in front, sometimes no. My soft preference would be without `The`, and my strong preference would be for consistency :)